ceph-salt-formula/salt/ceph-salt/apply/tuned-latency.sls

```diff
@@ -0,0 +1,29 @@
+
+{% if grains['id'] in pillar.get('ceph-salt:minions:latency', []) %}
+
```

Comment:
Please trigger `stage` events, so the user will see which stage is being executed in the UI, e.g.:

```
{{ macros.begin_stage('Configure latency tuned profile') }}
...
{{ macros.end_stage('Configure latency tuned profile') }}
```

Note that you will have to `{% import 'macros.yml' as macros %}` in the beginign of the file.